spec/controllers/sessions_controller_spec.rb

```diff
@@ -22,64 +34,66 @@
+        post :create, session: { email: @user.email, password: @user.password }
+      end
 
-    it { should redirect_to_url_after_create }
+      it { should redirect_to_url_after_create }
 
-    it 'sets the user in the clearance session' do
-      expect(controller.current_user).to eq @user
-    end
+      it "sets the user in the clearance session" do
+        expect(controller.current_user).to eq @user
+      end
 
-    it 'should not change the remember token' do
-      expect(@user.reload.remember_token).to eq 'old-token'
+      it "should not change the remember token" do
+        expect(@user.reload.remember_token).to eq "old-token"
+      end
     end
-  end
 
-  describe 'on POST to #create with good credentials and a session return url' do
-    before do
-      @user = create(:user)
-      @return_url = '/url_in_the_session?foo=bar'
-      @request.session[:return_to] = @return_url
-      post :create, session: { email: @user.email, password: @user.password }
-    end
+    context "with good credentials and a session return url" do
+      before do
+        @user = create(:user)
+        @return_url = "/url_in_the_session?foo=bar"
+        @request.session[:return_to] = @return_url
+        post :create, session: { email: @user.email, password: @user.password }
+      end
 
-    it 'redirects to the return URL' do
-      should redirect_to(@return_url)
+      it "redirects to the return URL" do
+        should redirect_to(@return_url)
+      end
     end
   end
 
-  describe 'on DELETE to #destroy given a signed out user' do
-    before do
-      sign_out
-      delete :destroy
-    end
-
-    it { should redirect_to_url_after_destroy }
-  end
+  describe "on DELETE to #destroy given a signed out user" do
+    context "given a signed out user" do
+      before do
+        sign_out
+        delete :destroy
+      end
 
-  describe 'on DELETE to #destroy with a cookie' do
-    before do
-      @user = create(:user)
-      @user.update_attribute :remember_token, 'old-token'
-      @request.cookies['remember_token'] = 'old-token'
-      delete :destroy
+      it { should redirect_to_url_after_destroy }
     end
 
-    it { should redirect_to_url_after_destroy }
+    context "with a cookie" do
+      before do
+        @user = create(:user)
+        @user.update_attribute :remember_token, "old-token"
+        @request.cookies["remember_token"] = "old-token"
+        delete :destroy
+      end
 
-    it 'should reset the remember token' do
-      expect(@user.reload.remember_token).not_to eq 'old-token'
-    end
+      it { should redirect_to_url_after_destroy }
 
-    it 'should unset the current user' do
-      expect(@controller.current_user).to be_nil
+      it "should reset the remember token" do
+        expect(@user.reload.remember_token).not_to eq "old-token"
+      end
+
+      it "should unset the current user" do
+        expect(@controller.current_user).to be_nil
+      end
     end
   end
 end
```

Comment:
@derekprior I have reorganized the spec file. It seemed to be using contexts in some places and not others.